docs/governance/kubernetes_update_process.md

```diff
@@ -0,0 +1,10 @@
+
+# Kubernetes versions update policy
+
+We will support 3 releases of Kubernetes, targeting the newest version as being the [default version in the GKE Rapid channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions) (since tests run on GKE we can't support versions that GKE doesn't support). However, we will ensure that at least one of the 3 versions chosen for each Agones release is supported by each of the major cloud providers (EKS and AKS). The vendored version of client-go will be aligned with the middle of the three supported Kubernetes versions.
```

Comment:
This paragraph is redundant to that which is in the docs, is it necessary to have and maintain two copies?